tool/processors/question/events/events.go

```diff
@@ -82,8 +83,14 @@ func monitorEvents(ctx *runtime.Context, config *data.Config) {
 			eventFormat = EventFormatPlainText
 			eventFormatDefaultOption = 2
 		}
+		retentionInDays := util.Choice("Log Group Retention in days", 1, []string{"-1", "1", "3", "5", "7", "14", "30", "60", "90", "120", "150", "180", "365", "400", "545", "731", "1827", "3653"})
```

Comment:
Is it really necessary to restrict the choices in this way?

Reply:
This is based on whats allowed on the [ cw logs api ](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html) but I believe that the user should be allowed more flexibility and their looking into it

Reply:
Right, but validation of these values isn't done anywhere else in the code.

Reply:
I had written the `isValidRetention` function for that purpose but I'm realizing I forgot to use it in the `knownConfigKeys.go`

Reply:
I'd prefer abstracting away this so that we don't parrot what the "correct" values are here, since they are subject to change. If there's another place where we actually perform the validation, this logic should be in there.